### PR TITLE
travis: test building a package

### PR DESCRIPTION
**Pull request by johnelse**

### CODE REVIEW

File: test/travis-test-internal.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -eux
+
+ulimit -s 16384
```
Comment (robhoes) [inline test/travis-test-internal.sh:5]: I normally do `ulimit -s unlimited`, just in case :)

Comment (johnelse) [inline test/travis-test-internal.sh:5]: datamodel.ml is only gonna get bigger :wink:

**Comment by johnelse** [comment]
This needs updating to build from the SRPM rather than the git repo.

**Comment by jonludlam** [comment]
@johnelse Yes, the SRPM build is the way to go - this prevents things being out-of-sync - we _know_ that the SRPM in the repo should build correctly, whereas HEAD of xen-api.git might have moved in in an incompatible way.

**Comment by johnelse** [comment]
Hmm, the xapi SRPM doesn't actually build - see https://travis-ci.org/xenserver/xenserver-build-env/builds/101022186

I've changed it to build ocaml-xcp-idl for simplicity (and speed :-) )

File: test/travis-test-internal.sh (hunk 1 above)
Comment (robhoes) [inline test/travis-test-internal.sh:5]: Hence "unlimited" ;)

**Pull request merged**